src/test/regress/sql/gp_parallel.sql

```diff
@@ -510,6 +510,84 @@ abort;
 -- End of Test locus after eliding mtion node.
 --
 
+--
+-- Parallel Refresh AO Materialized View
+--
+create or replace function refresh_compare(ao_row bool, verbose bool, OUT parallel_is_better bool) as $$
+declare
+ t timestamptz;
+ dur0 interval;
+ dur1 interval;
+ results0 RECORD;
+ results1 RECORD;
+begin
+  create table t_p(c1 int, c2 int) with(parallel_workers=8) distributed by(c1);
+  insert into t_p select i, i+1 from generate_series(1, 10000000)i;
+  analyze t_p;
+  if ao_row then
+    create materialized view matv using ao_row as select sum(a.c2) as c2, avg(b.c1) as c1 from t_p a join t_p b on a.c1 = b.c1 with no data distributed by(c2);
+  else
+    create materialized view matv using ao_column as select sum(a.c2) as c2, avg(b.c1) as c1 from t_p a join t_p b on a.c1 = b.c1 with no data distributed by(c2);
+  end if;
+  set optimizer = off;
+  -- refresh
+  set enable_parallel=off;
+  t = clock_timestamp();
+  refresh materialized view matv;
+  dur0 = age(clock_timestamp(), t);
+  select * into results0 from matv;
+  if refresh_compare.verbose then
+    raise notice 'Non-parallel refresh duration=%', dur0;
+    raise notice 'Non-parallel results=%', results0;
+  end if;
+  -- parallel refresh
+  set enable_parallel=on;
+  t = clock_timestamp();
+  refresh materialized view matv;
```

Comment:
support refresh concurrent in parallel mode ?

Reply:
Parallel has no impact on that if mat view  is supported to be concurrently refresh
But AO doesn't have an Unique index now, so it couldn't refresh concurrently on AO mat view. 
Maybe it could be enabled after #68